api/xds_protocol.rst

```diff
@@ -75,20 +64,20 @@ API flow
 ~~~~~~~~
 
 For typical HTTP routing scenarios, the core resource types for the client's configuration are
-`Listener`, `RouteConfiguration`, `Cluster`, and `ClusterLoadAssignment`. Each `Listener` resource
-may point to a `RouteConfiguration` resource, which may point to one or more `Cluster` resources,
-and each `Cluster` resource may point to a `ClusterLoadAssignment` resource.
+``Listener``, ``RouteConfiguration``, ``Cluster``, and ``ClusterLoadAssignment``. Each ``Listener`` resource
```

Comment:
these are kinda inconsistent - in the rest of the docs it uses links to the api protos

Reply:
It would be nice to update, sure, but not compulsory.

Reply:
cool - im gonna pass for now - it feels like not an immediate priority